paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -133,7 +131,7 @@ internal class FormViewModel @Inject internal constructor(
         combine(
             hiddenIdentifiers,
             elementsFlow.map {
-                it ?: emptyList()
+                it ?: emptyList() // TODO(warnings)
```

Comment:
👀

Reply:
Just making sure you're paying attention 😉 🙈  LOL, nice catch.